src/files/BrsFile.ts

```diff
@@ -1369,7 +1631,7 @@ export class BrsFile {
     /**
      * Convert the brightscript/brighterscript source code into valid brightscript
      */
-    public transpile(): CodeWithSourceMap {
+    public transpile(scope?: Scope): CodeWithSourceMap {
```

Comment:
You're not using scope in this function at all, so does the parameter need to be there?